experiments/golden-results/StratoX-summary.txt

```diff
@@ -2110,7 +2115,7 @@ Error detected at REDACTED
 --
 Occurs: 1 times
 +===========================GNAT BUG DETECTED==============================+
-| GNU Ada (ada2goto) Assert_Failure failed precondition from range_check.ads:55|
+| GNU Ada (ada2goto) Assert_Failure failed precondition from range_check.ads:56|
```

Comment:
Please could you look at this, I am 99% sure it is a missing case in an precondition.  It is fine to add it and then throw an unsupported feature in the implementation of `Make_Range_Assert_Expr` if it is a lot of effort to support.